tests/testthat/test-add_co2-summarize_co2_range.R

```diff
@@ -0,0 +1,117 @@
+test_that("different benchmarks output different number of rows", {
```

Comment:
This means that downstream from this function, outputs will have a more complex structure than before. No more we can expect, say, 1 or 4 rows. Instead the number of rows depends on the benchmark.